genny-project/layouts
# Pcs 90 part4

# Issue https://outcomelife.atlassian.net/browse/PCS-226
# Description:
Change bucket-view look, added legends for every bucket

# How To Test
1. Go to PCSS.
2. Select a property in the table-view and goto "Process View"
3. You should see legends in every bucket with status colour mapping
4. The spacing between buckets lessened, the colour of spacing between tabs changed
